app/report/utils.py

```diff
@@ -128,3 +132,41 @@ def generate_csv_from_notifications(service_id, notification_type, language, day
     query = build_notifications_query(service_id, notification_type, language, days_limit)
     copy_command = compile_query_for_copy(query)
     stream_query_to_s3(copy_command, s3_bucket, s3_key)
+
+
+def send_requested_report_ready(report) -> None:
+    """
+    We are sending a notification to the user to inform them that their requested
+    report is ready.
+    """
+    template = dao_get_template_by_id(current_app.config["REPORT_DOWNLOAD_TEMPLATE_ID"])
+    service = Service.query.get(current_app.config["NOTIFY_SERVICE_ID"])
+    report_service = Service.query.get(report.service_id)
+
+    if template.template_type == "email":
+        report_name_en = f"{report.requested_at.date()}-emails-{report_service.name}"
+        report_name_fr = f"{report.requested_at.date()}-courriels-{report_service.name}"
+    else:
+        report_name_en = f"{report.requested_at.date()}-sms-{report_service.name}"
+        report_name_fr = f"{report.requested_at.date()}-sms-{report_service.name}"
+
+    saved_notification = persist_notification(
+        template_id=template.id,
+        template_version=template.version,
+        recipient=report.requesting_user.email_address,
+        service=service,
+        personalisation={
+            "name": report.requesting_user.name,
+            "report_name": report_name_en,
```

Comment:
```suggestion
            "report_name_en": report_name_en,
```

Reply:
I think this is messing up the email variable:
<img width="635" alt="Screenshot 2025-04-16 at 2 21 37 PM" src="https://github.com/user-attachments/assets/b291e8a6-4012-4108-ad59-250c7545f777" />